### PR TITLE
#22 Updated dependencies

### LINKED ISSUE #19: refactor bucketfs-document-files to use newest version of common-document-files
Bucketfs-document-files hasn't had a major document-files version bump yet. It seems the architectural design of common-document-files changed quite a bit since then and this 'dialect' version needs a rework/refactoring as well.

### LINKED ISSUE #22: Fix vulnerabilities
* [ch.qos.reload4j:reload4j:jar:1.2.18.3](https://ossindex.sonatype.org/component/pkg:maven/ch.qos.reload4j/reload4j@1.2.18.3?utm_source=ossindex-client&utm_medium=integration&utm_content=1.8.1) in compile
    * [sonatype-2022-5401](https://ossindex.sonatype.org/vulnerability/sonatype-2022-5401): 1 vulnerability (8.6)

### PR DESCRIPTION
**Pull request by kaklakariada**
Closes #22
fixes #19 
Closes https://github.com/exasol/virtual-schema-common-document-files/issues/109

### CODE REVIEW

File: src/test/java/com/exasol/adapter/document/documentfetcher/files/BucketFsFileContentTest.java [hunk 1]
```diff
@@ -49,17 +55,17 @@ void getInputStreamFailsForMissingFile() {
...
     }
 
     @Test
     void loadAsyncFailsForMissingFile() {
         final Path file = this.tempDir.resolve("non-existing-file");
         final Future<byte[]> content = create(file).loadAsync();
         final ExecutionException exception = assertThrows(ExecutionException.class, () -> content.get());
-        assertThat(exception.getMessage(), equalTo("java.lang.IllegalStateException: F-BFSVS-6: Could not open " + file
-                + ". This is an internal error that should not happen. Please report it by opening a GitHub issue."));
+        assertThat(exception.getMessage(), equalTo("java.lang.IllegalStateException: F-BFSVS-6: Could not open '" + file
```
Comment (ckunki) [inline src/test/java/com/exasol/adapter/document/documentfetcher/files/BucketFsFileContentTest.java:67]: Maybe we can also Rename error codes from BFSVS to VSBFS in this occasion? 
see issue #23

Comment (kaklakariada) [reply src/test/java/com/exasol/adapter/document/documentfetcher/files/BucketFsFileContentTest.java:67]: ✅

File: doc/changes/changes_1.0.0.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Virtual Schema for document data on BucketFS 1.0.0, released 2022-09-29
+
+Code name: Upgrade virtual-schema-common-document-files
+
+## Summary
+
+In this release we upgraded version 7.1.1 of `virtual-schema-common-document-files`, which means that it now uses the new [connection parameter specification](https://github.com/exasol/connection-parameter-specification/blob/main/specification.md). This means you need to update the connection, see the [user guide](https://github.com/exasol/bucketfs-document-files-virtual-schema/blob/main/doc/user_guide/user_guide.md#creating-a-connection) for details.
```
Comment (ckunki) [inline doc/changes/changes_1.0.0.md:7]:
```suggestion
This release upgraded to version 7.1.1 of `virtual-schema-common-document-files` using the new [connection parameter specification](https://github.com/exasol/connection-parameter-specification/blob/main/specification.md). Users of Virtual Schema for document data on BucketFS therefore must update their connection, see the [user guide](https://github.com/exasol/bucketfs-document-files-virtual-schema/blob/main/doc/user_guide/user_guide.md#creating-a-connection) for details.
```

File: doc/changes/changes_1.0.0.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+In previous versions you could configure a base path for the connection. Now the virtual schema only uses bucket `/buckets/bfsdefault`. If you need this configuration, please vote for [issue #25](https://github.com/exasol/bucketfs-document-files-virtual-schema/issues/25).
```
Comment (ckunki) [inline doc/changes/changes_1.0.0.md:9]:
```suggestion
Previous versions of Virtual Schema for document data on BucketFS allowed to configure a base path for the connection while the current release only uses bucket `/buckets/bfsdefault`. If you need this configuration, please vote for [issue #25](https://github.com/exasol/bucketfs-document-files-virtual-schema/issues/25).
```

File: doc/changes/changes_1.0.0.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+We also fixed [sonatype-2022-5401](https://ossindex.sonatype.org/vulnerability/sonatype-2022-5401) in reload4j.
```
Comment (ckunki) [inline doc/changes/changes_1.0.0.md:11]:
```suggestion
This release also fixes [sonatype-2022-5401](https://ossindex.sonatype.org/vulnerability/sonatype-2022-5401) in reload4j.
```

**Review by ckunki — CHANGES_REQUESTED** [review]

**Review by ckunki — APPROVED** [review]

**Pull request merged**